Source/ACE.Server/Managers/RecipeManager.cs

```diff
@@ -112,7 +113,8 @@ public static void UseObjectOnTarget(Player player, WorldObject source, WorldObj
 
                     if (destroySource)
                     {
-                        if (source.OwnerId == player.Guid.Full)
+                        // TODO - Check if the source is wielded and handle appropriately
```

Comment:
A new function was recently added for this: TryRemoveItemWithNetworking
It checks if the item is currently wielded first, and if so, tries to unwield it before calling TryRemoveFromInventoryWithNetworking

Reply:
This function does not remove enchantments applied by the item. Is that intentional, or oversight? I'm going to guess oversight, since I see nothing in the functions that use it that would handle this, either.

Reply:
Not removing enchantments - oversight, although this should be handled in the UnwieldItemWithNetworking function

Placing in backpack first - this is known, and was a limitation of the current TryRemoveFromInventoryWithNetworking system